services/workflows-service/src/alert/alert.controller.external.ts

```diff
@@ -135,7 +137,17 @@ export class AlertControllerExternal {
   async decision(
     @common.Body() { alertIds, decision }: AlertDecisionDto,
     @CurrentProject() currentProjectId: TProjectId,
+    @UserData() authenticatedAssignee: AuthenticatedEntity,
   ): Promise<TBulkAssignAlertsResponse> {
+    // Assign alerts to the authenticated assignee
+    if (authenticatedAssignee?.user?.id) {
```

Comment:
Is it something that you talked with @nitzanballerine  that once you take a decision you assign the task to yourself?

i think as we have assignedby & at lets add additional properties for the person who did the decision.

because once the admin take an action and set a decision you will re-assign yourself

Reply:
I brought your attention to the origin of the ID. This requirement is in my task description and was just confirmed by Nitzan. Note, you also don't need to be assigned to make a decision on an alert.

Reply:
did we discuss about it?